config/overlays/odh/inferenceservice-config-patch.yaml

```diff
@@ -5,6 +5,14 @@ metadata:
   namespace: kserve
 data:
   explainers: "{}"
+  oauthProxy: |- 
+    {
+      "image" : "$(oauth-proxy)",
+      "memoryRequest": "256Mi",
```

Comment:
Guess this values will fit it better:

```
          requests:
            memory: "64Mi"
            cpu: "100m"
          limits:
            memory: "128Mi"
            cpu: "200m"
```

256 is too much memory for oauth proxy